engine/engine/content/builtins/scripts/edn.lua

```diff
@@ -14,34 +20,15 @@ local escape_char_map = {
   ["\t" ] = "\\t",
 }
 
-local function raw_tostring(val)
-  local old_mt = debug.getmetatable(val)
-  debug.setmetatable(val, nil)
-  local raw_str = tostring(val)
-  debug.setmetatable(val, old_mt)
-  return raw_str
-end
-
-local function has_custom_tostring(val)
```

Comment:
Merged into a function where it's used to reduce count of `getmetatable()` or `debug.getmetatable()` calls